components/kyma-environment-broker/internal/provider/azure_provider.go

```diff
@@ -144,8 +146,11 @@ func (p *AzureTrialInput) ApplyParameters(input *gqlschema.ClusterConfigInput, p
 		}
 	}
 
-	if params.Region != nil {
+	if params.Region != nil && *params.Region != "" {
+		fmt.Println(*params.Region)
 		updateString(&input.GardenerConfig.Region, toAzureSpecific[*params.Region])
+		fmt.Println(input.GardenerConfig.Region)
+		fmt.Println(toAzureSpecific[*params.Region])
```

Comment:
nit: are these `fmt.Println` statements intended to stay?

Reply:
removed